modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -1,59 +1,62 @@
 <template>
-  <transition name="fade-stage">
-    <div
-      class="apos-login apos-theme-dark apos-theme--primary-purple"
-      v-show="loaded"
-    >
-      <div class="apos-login__wrapper">
-        <transition name="fade-body">
-          <div class="apos-login__upper" v-show="loaded">
-            <div class="apos-login__header">
-              <label
-                class="apos-login__project apos-login__project-env"
-                :class="[`apos-login__project-env--${context.env}`]"
-              >
-                {{ context.env }}
-              </label>
-              <label class="apos-login__project apos-login__project-name">
-                {{ context.name }}
-              </label>
-              <label class="apos-login--error">
-                {{ error }}
-              </label>
-            </div>
+  <div ref="login">
```

Comment:
needed to wrap this in something .. an element's `$ref` are not available while it's hidden (pre transition) so adding it inside the transition was not working (and lost a bit of time figuring it)

Reply:
gone now